server/routes/index.ts

```diff
@@ -29,9 +29,16 @@ export default function routes(router: Router, services: Services): Router {
   const referralsController = new ReferralsController(services.interventionsService, services.communityApiService)
 
   get('/', (req, res, next) => {
-    res.render('pages/index')
+    const { authSource } = res.locals.user
+    if (authSource === 'delius') {
```

Comment:
Optional, but I wonder if this behaviour is something that might be better extracted to a middleware function (like `authorizationMiddleware` to keep this logic out of the routes file?

Reply:
Good question, I'm not sure where's the best place for it. I guess that `authorizationMiddleware` acts on all requests and is designed to block the user from performing certain actions if they shouldn't be allowed to. Whereas all that this code is doing is just deciding where `/` takes a given user, which I guess I thought of as "routing". But not sure. Perhaps we could leave it here for now.

Reply:
Yeah, we can leave it here for now. Sorry, if it wasn't clear, I meant using `authorizationMiddleware` as an example, rather than putting this behaviour in there, we'd probably create a new middleware. I guess in future we could use something like that to stop service providers being able to access the "make a referral" & "monitor" flows, but it's not something we've really thought about yet (and not necessarily relevant to this logic that directs a user to the correct homepage)